opm/simulators/aquifers/AquiferCarterTracy.hpp

```diff
@@ -117,12 +133,16 @@ class AquiferCarterTracy : public AquiferInterface<TypeTag>
     inline void calculateEqnConstants(Scalar& a, Scalar& b, const int idx, const Simulator& simulator)
```

Comment:
`a` and `b` can be returned as a `std::pair()`

Reply:
> `a` and `b` can be returned as a `std::pair()`

I've pushed an update to make it so.